tests/unit/test_models/test_full_battery_models/test_base_battery_model.py

```diff
@@ -430,7 +430,7 @@ def test_default_solver(self):
         assert isinstance(model.default_solver, pybamm.CasadiSolver)
 
         # check that default_solver gives you a new solver, not an internal object
-        solver = model.default_solver
+        _ = model.default_solver
         solver = pybamm.BaseModel()
         assert isinstance(model.default_solver, pybamm.CasadiSolver)
         assert isinstance(solver, pybamm.BaseModel)
```

Comment:
I could be wrong, but I think the point of this test was to check if the `solver` variable is being overwritten correctly? In that case we can ignore the error that the linter is throwing here; otherwise, this looks good!

PS: I don't really get what this test is trying to do.